sktime/forecasting/base/convertIO/_convertIO.py

```diff
@@ -0,0 +1,190 @@
+# -*- coding: utf-8 -*-
+"""
+Machine type converters for scitypes
+
+Exports
+-------
+convert : dictionary of functions, with signature as below
+        indexed by triples of types
+         1st element = convert from - type
+         2nd element = convert to - type
+         3rd element = considered as this scitype - string
+        elements are conversion functions of machine type (1st) -> 2nd
+
+convert_to : function/dispatch version of convert, with 1st elt inferred
+
+
+-----------------------------------------
+
+Function signature of
+convert[(from_type, to_type, as_scitype)]
+
+Parameters
+----------
+what : from_type - object to convert
+store : reference of storage for lossy conversions, default=None (no store)
+
+Returns
+-------
+converted_what : to_type - object what converted to to_type
+
+Raises
+------
+ValueError and TypeError, if requested conversion is not possible
+                            (depending on conversion logic)
+
+-----------------------------------------
+
+Function signature of convert_to
+
+Parameters
+----------
+what : object to convert (any type)
+to_type : type, the type to convert "what" to
+as_scitype : str - name of scitype the object "what" is considered as
+store : reference of storage for lossy conversions, default=None (no store)
+
+Returns
+-------
+converted_what : to_type - object what converted to to_type
+
+Raises
+------
+TypeError if no suitable key in convert exists
+    i.e., (type(what), to_type, as_scitype)
+
+
+copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""
+
+
+__author__ = ["fkiraly"]
+
+
+import numpy as np
+import pandas as pd
+
+
+# dictionary indexed by triples of types
+#  1st element = convert from - type
+#  2nd element = convert to - type
+#  3rd element = considered as this scitype - string
+# elements are conversion functions of machine type (1st) -> 2nd
+
+convert = dict()
+
+
+def convert_identity(what, store=None):
```

Comment:
minor preference for just `identity`. `convert_identity` sounds like identity is being converted.

But I realise `identity` is not a verb... I feel like `identity` is such a standard name of a function that the verb convention can be ignored in this instance

Reply:
well, it's not the identity function, it has two arguments

Reply:
also, it's private, no one sees it except if you specifically look for it